packages/hash/frontend/src/pages/login.page.tsx

```diff
@@ -108,13 +111,28 @@ const LoginPage: NextPageWithLayout = () => {
             identifier: email,
             password,
           })
-          // We logged in successfully! Let's bring the user home.
-          .then(() => {
+          // We logged in successfully! Let's redirect the user.
+          .then(async () => {
```

Comment:
Would the result of the push at line 105 ever be able to cancel the executiong of the code here? Or can the code be assumed to run no matter than route the user is in?

Reply:
> Would the result of the push at line 105 ever be able to cancel the executing of the code here? 

My understanding is no it wouldn't, as it's a shallow redirect.